packages/gatsby/src/bootstrap/load-plugins/load.js

```diff
@@ -81,7 +84,11 @@ function resolvePlugin(pluginName) {
    * which should be located in node_modules.
    */
   try {
-    const resolvedPath = slash(path.dirname(require.resolve(pluginName)))
+    const requireSource =
+      rootDir !== null
+        ? createRequireFromPath(`${rootDir}/:internal:`)
+        : require
```

Comment:
can there be cases where `rootDir` is falsy?

Reply:
I'm not aware of any - I originally just wanted to make sure to preserve BC if something was calling the exported function with a single argument.